.travis.yml

```diff
@@ -16,19 +14,13 @@ jobs:
       name: Lint JavaScript
     - script: yarn test
       name: Tests
-    # build only for master and devel (tmp: "improvement/semantic-release").
-    - stage: build
-      if: branch IN ("master", "devel")
+    - stage: Build & Publish
+      if: branch = master
       script:
         - yarn
         - yarn build
-    # publish for master only thanks to semantic-release
-    - stage: publish
-      if: branch IN ("master") AND type != pull_request
```

Comment:
I thought we have to specify `type != pull request` to avoid triggering release on PR test but it seems it's not required: https://travis-ci.com/github/ForestAdmin/forest-express/builds/163902993/config

I don't remember why I added this line though.